src/Client/Updater.php

```diff
@@ -138,6 +139,26 @@ public function validateTarget($targetRepoPath, $targetStream)
         return true;
     }
 
+    /**
+     * Converts a metadata timestamp string into an immutable DateTime object.
+     *
+     * @param string $timestamp
+     *     The timestamp string in the metadata.
+     *
+     * @return \DateTimeImmutable
+     *     An immutable DateTime object for the given timestamp.
+     *
+     * @throws FormatException
+     *     Thrown if the timestamp string format is not valid.
+     */
+    protected function metadataTimestampToDatetime(string $timestamp) : \DateTimeImmutable
```

Comment:
I think `Datetime` should be `DateTime`  because this is 2 words and how php functions do this.

Reply:
Agreed.

Reply:
Pushed a commit to fix this.